loadtest/user/userentity/actions.go

```diff
@@ -242,6 +242,15 @@ func (ue *UserEntity) GetChannel(channelId string) error {
 	return ue.store.SetChannel(channel)
 }
 
+func (ue *UserEntity) GetChannelsForTeam(teamId, userId string) error {
```

Comment:
This method should only need a `teamId` since the `userId` should be the one of the current user and should be taken from the internal state.

Reply:
Wait, this is the same as my PR https://github.com/mattermost/mattermost-load-test-ng/pull/58. We should probably make the method name the same as the underlying client API name to avoid confusion.

Reply:
We applied the same convention to previous implementations and I would keep it that way. The idea is that we add the suffix `ForUser` when we need to specify the `userId`. There are cases where we want to have them both and one would wrap the other to simplify usage. An example is `GetProfileImage` and `GetProfileImageForUser`.

Reply:
Changed :)